renovate.json

```diff
@@ -24,13 +24,11 @@
     },
     {
       "groupName": "MUI public packages",
-      "matchPackageNames": ["@mui/*", "!@mui/internal-*", "!@mui/docs"],
       "allowedVersions": "!/-dev/"
     },
     {
-      "matchDepTypes": ["devDependencies"],
```

Comment:
My hunch is also the same - that this group is breaking the `playwright` bump. 🙈